libs/common/mem_operations.hpp

```diff
@@ -27,21 +26,25 @@ namespace iroha {
 
 #ifdef __linux__
   inline uint64_t getMemoryUsage() {
-    auto parseLine = [](char *line) {
-      while (*line >= '0' && *line <= '9') ++line;
+    auto parseLine = [](char const *line) {
+      while (!std::isdigit(*line)) ++line;
```

Comment:
Thanks for changing as suggested, but isn't the behaviour opposite?
1. 
```
while (*line >= '0' && *line <= '9')
```
the loop is working when characters are between [0, 9]
2. after changing:
```
while (!std::isdigit(*line))
```
the loop is working for all characters expect for [0, 9]
____
So shouldn't be without `!`?:
```
while (std::isdigit(*line)) ++line;
```

Reply:
It was a bug, while refactoring. Now it's correct:)